rend3/src/managers/skeleton.rs

```diff
@@ -48,34 +69,40 @@ impl SkeletonManager {
         handle: &SkeletonHandle,
         skeleton: Skeleton,
     ) {
-        let vertex_range = mesh_manager.allocate_skeleton_mesh(device, queue, encoder, &skeleton.mesh);
+        let mesh_range = mesh_manager.internal_data(skeleton.mesh.get_raw()).vertex_range.clone();
+        let skeleton_range = mesh_manager.allocate_skeleton_mesh(device, queue, encoder, &skeleton.mesh);
+
+        let input = GpuVertexRanges {
+            skeleton_range: [skeleton_range.start as u32, skeleton_range.end as u32],
+            mesh_range: [mesh_range.start as u32, mesh_range.end as u32],
+        };
```

Comment:
This is cached on the `InternalSkeleton` now to avoid having to fetch the mesh and its range in a hot loop (pre-skinning)

Reply:
This is the right way to go about it, but we also need to properly invalidate it when vertex buffers reallocate. #332 is just this but with objects and indices.